source/index.txt

```diff
@@ -95,11 +117,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Learn
+------
+
+Visit the Developer Hub and MongoDB University to learn more about the
+MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub provides tutorials and social engagement for
+developers.
+
+To learn how to use MongoDB features with the Java driver, see the `How
+To's and Articles page <https://www.mongodb.com/developer/learn/?content=Articles&text=java#main>`__.
+
+To learn what fellow developers are asking about, experiencing or their
+interests in regards to the Java driver, see the `forums page
+<https://www.mongodb.com/community/forums/tag/java>`__.
+
+MongoDB University
+~~~~~~~~~~~~~~~~~~
+
+MongoDB University provides free courses to teach everyone how to use
+MongoDB.
 
 Take the free online course taught by MongoDB
```

Comment:
I wonder if "taught by instructors at MongoDB" might be more accurate and informative.